tools/__init__.py

```diff
@@ -228,6 +229,12 @@ def _package_str(self):
             ver_str = self.package
         return ver_str
 
+    def is_attempted(self):
+        return self._is_attempted
+
+    def is_installed(self):
```

Comment:
Whoops that is a mistake on my behalf. I think I was running into problems before where I wasn't calling `install_and_get_path()`, which resulted in problems. I still might have to make a change on `verify_install` to allow optionally skipping executability check because krona doesn't have a `krona` executable.